scripts/ci/libraries/_initialization.sh

```diff
@@ -240,9 +240,9 @@ function get_environment_for_builds_on_ci() {
     export CI_BUILD_ID="0"
     export CI_JOB_ID="default-job-id"
     if [[ ${CI:=} != "true" ]]; then
-        echo
-        echo "This is not a CI environment!. Staying with the defaults"
-        echo
+        print_info
+        print_info "This is not a CI environment!. Staying with the defaults"
```

Comment:
```suggestion
        print_info "This is not a CI environment! Staying with the defaults"
```

Reply:
Should we stick to the interpunction in messages? Sometimes at the end is full stop, sometimes doesn't.